crates/nu-command/Cargo.toml

```diff
@@ -101,7 +118,14 @@ uu_mv = { workspace = true, optional = true }
 uu_touch = { workspace = true, optional = true }
 uu_uname = { workspace = true, optional = true }
 uu_whoami = { workspace = true, optional = true }
-uuid = { workspace = true, features = ["v4"], optional = true }
+uuid = { workspace = true, features = [
+	"v1",
+	"v3",
+	"v4",
+	"v5",
+	"v7",
+	"fast-rng",
```

Comment:
As a guardian of the dependency tree: Do we need the `fast-rng` feature?

Reply:
my bad I'll remove that right away.